src/cascadia/TerminalApp/TabRowControl.xaml

```diff
@@ -34,6 +34,17 @@ the MIT License. See LICENSE in the project root for license information. -->
                 BorderThickness="0"
                 CornerRadius="{Binding Source={ThemeResource OverlayCornerRadius}, Converter={StaticResource TopCornerRadiusFilterConverter}}"
                 AutomationProperties.AccessibilityView="Control">
+                <ToolTipService.ToolTip>
+                    <ToolTip x:Name="TabTip"

```

Comment:
this one doesn't need a name since we won't be addressing it directly~!

Reply:
I refer to it now so we can reuse it for the profiles in the dropdown

Reply:
Okay wait now the tooltip on the '+' doesn't appear anymore

Reply:
Okay fixed now